persistent/tests/test_timestamp.py

```diff
@@ -100,9 +100,10 @@ def test_ctor_from_string(self):
         self.assertEqual(ts.timeTime(), DELTA_SECS)
 
     def test_ctor_from_string_non_zero(self):
-        before = self._makeOne(2011, 2, 16, 14, 37, 22.0)
+        before = self._makeOne(2011, 2, 16, 14, 37, 22.80544)
         after = self._makeOne(before.raw())
         self.assertEqual(before.raw(), after.raw())
+        self.assertEqual(before.timeTime(), 1297867042.80544)
```

Comment:
There's `time.mktime` but it's for local times. On my machine (GMT+1):
```python
>>> time.mktime((2011, 2, 16, 14, 37, 22, 0, 0, 0)) - 1297867042
3600
```
(with dummy values for `tm_wday` and `tm_yday`)

I don't find such expected values problematic, when the alternative is to duplicate the tested code. And in case where hardcoding values in the code is unmaintainable, I prefer a mechanism with « dumps » (the idea is to have a mode in which the test suite generates dumps instead of comparing values against them). I have started to do for a big test suite (45k lines excluding dumps) because I was tired of implementing many features twice.

```
# date -u -d @1297867042
Wed Feb 16 14:37:22 UTC 2011
```